packages/perseus-editor/src/widgets/deprecated-standin-editor.tsx

```diff
@@ -0,0 +1,29 @@
+import {EditorJsonify} from "@khanacademy/perseus";
+import * as React from "react";
+
+type Props = {
+    // Callback for when a widget prop is changed.
+    onChange: (options: any) => void;
+};
+
+class DeprecatedStandinEditor extends React.Component<Props> {
+    static widgetName = "deprecated-standin" as const;
+
+    serialize(): any {
+        return EditorJsonify.serialize.call(this);
+    }
+
+    render(): React.ReactNode {
+        return (
+            <div>
+                <p>This widget has been deprecated and removed</p>
+                <p>
+                    Learners will see a message and they will not be graded on
+                    this part
+                </p>
```

Comment:
**thought**: Would we want to provide further guidance here?  Perhaps "Please remove this widget (or replace it with a supported one) from this exercise and publish" ?